### PR TITLE
ECHO-1147: Generalization Asset-API

### PR DESCRIPTION
**Pull request by Ashes17b**

### CODE REVIEW

File: api-reference/echo-node-api/asset-api.md [hunk 1]
```diff
@@ -1,39 +1,144 @@
-Asset API
```
Comment (pixelplex) [inline api-reference/echo-node-api/asset-api.md:1]: `APIs` are an extra header. The top-level header should be an `Asset API`.

File: api-reference/echo-node-api/asset-api.md [hunk 2]
```diff
@@ -1,39 +1,144 @@
...
-=========
+# APIs
 
-### get_asset_holders(string asset_id, int start, int limit)
+## Asset API
+
+---
```
Comment (pixelplex) [inline api-reference/echo-node-api/asset-api.md]: i think it's unnecessary (I'm about the line)

File: api-reference/echo-node-api/asset-api.md (hunk 1 above)
Comment (Ashes17b) [reply api-reference/echo-node-api/asset-api.md:1]: Is this about all api?  And database, login too?

File: api-reference/echo-node-api/asset-api.md [hunk 3]
```diff
@@ -1,39 +1,144 @@
...
-- *asset_id* ID of the asset to retrieve, a string that look like 1.6.<id-of-the-asset>
-- *start* The acount id to start retrieving from
-- *limit* Limit to that much accounts (max 100)
+##### Parameters
 
-#### Returns
-An array of information about asset holders, that has the following structure:
+| Option                   | Description                                                                |
+|--------------------------|:---------------------------------------------------------------------------|
+| `asset_id_type asset_id` | ID of the asset to retrieve, a string that look like 1.6.<id-of-the-asset> |
```
Comment (pixelplex) [inline api-reference/echo-node-api/asset-api.md]: maybe 1.3.x ?

**Pull request merged**